changelog/66-add.md

```diff
@@ -0,0 +1,25 @@
+---
+issue: '#66'
+type: add
+audience: user
+date: '2020-05-13'
+components:
+  - cli
+---
+# Add cli tool to generate change logs
+
+Added a cli tool that can generate change log templates based on the provided snage config.
+The generated template will be in the `Front-Matter` format consisting of a YAML header with meta
+data and a markdown body.
+
+Can be called via `yarn dev create`. Requires a valid config file that can be provided via
+`--config path/to/your/config/.snage.yaml`. Defaults to `.snage.yaml`. Every field
+defined in your snage config can be added as params to the `create` call and will be
+set in the generated `.md-file`.
+
+Per default `--interactive` is enabled, meaning the tool will ask for input for every field not 
+provided as param while calling `create`. Can be disabled with `--no-interactive`. In that case
+required fields with no values will be added as comments and optional fields won't be added.
+
+For more information and description of all available params call `yarn dev create --help`; 
```

Comment:
In präsens schreiben. 

Ich glaub hier macht es nicht viel sinn nochmal die struktur von den markdown files zu erkláren. Hier hat sich nichts geändert, das war auch vorher schon so. 
Zudem sollten nicht unsere dev commands genutzt werden, das hier sind user release notes nicht dev release notes.

Hier mein Vorschlag:
````suggestion
`snage create` creates change log notes. Field values can either be added via parameters or will be ask interactively.
The latter can be disabled by adding the `--no-interactive` flag.

Available parameters and descriptions can be shown via `snage create --help`.
````